src/System.Management.Automation/FormatAndOutput/common/BaseFormattingCommand.cs

```diff
@@ -262,6 +262,9 @@ private void ProcessObject(PSObject so)
             }
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.1.80', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 257
257:             $result[0].TargetAddress | Should -BeExactly $realAddress
```